wolfssl/test.h

```diff
@@ -846,7 +847,8 @@ static INLINE void tcp_accept(SOCKET_T* sockfd, SOCKET_T* clientfd,
     #if defined(_POSIX_THREADS) && defined(NO_MAIN_DRIVER) && !defined(__MINGW32__)
         /* signal ready to tcp_accept */
         {
```

Comment:
Please cleanup the extra braces {} that are no longer required at 849 and 859. Looks good tho. Thanks!